### ISSUE TITLE
welcome email to all learners fails with `RequestError: TransportError`

### ISSUE THREAD

**Issue by pdpinch** [mitodl/micromasters#2513]
https://sentry.io/mit-office-of-digital-learning/micromasters/issues/214710346/

```
RequestError: TransportError(400, 'search_phase_execution_exception', 'Expected [FIELD_NAME] under a [START_OBJECT], but got a [START_OBJECT] in [courses]')
(17 additional frame(s) were not displayed)
...
  File "rest_framework/views.py", line 474, in dispatch
    response = handler(request, *args, **kwargs)
  File "mail/views.py", line 49, in post
    filter_on_email_optin=True
  File "search/api.py", line 107, in prepare_and_execute_search
    return search_func(search_obj)
  File "search/api.py", line 131, in get_all_query_matching_emails
    search_results = execute_search(search_obj[from_index: to_index])
  File "search/api.py", line 29, in execute_search
    return search_obj.execute()

RequestError: TransportError(400, 'search_phase_execution_exception', 'Expected [FIELD_NAME] under a [START_OBJECT], but got a [START_OBJECT] in [courses]')
```

**Comment by pdpinch** [comment]
This works: filtering the query down to one user and messaging that one user. 

This also works (on micromasters-rc): sending a message with no filter. There are less than 10 learners in that program on micromasters-rc. 

One theory: it's a bug in the search pagination where there is a large number of learners to send the message to.

**Comment by pdpinch** [comment]
From Anna:

> No filters - I was trying to send a general welcome email to all students.

**Comment by pdpinch** [comment]
closed by #2518
